NetworkingDemo/DemoRepository.swift

```diff
@@ -30,7 +30,7 @@ internal class DemoRepository: AbstractRepository {
     }
     
     public func noAnswerEntities(userID: Int) -> SignalProducer<Void, RepositoryError> {
-        let path = DemoRepository.ReadPrefixPath + "/\(userID)" + DemoRepository.ReadSuffixPath
+        let path = DemoRepository.ReadPrefixPath + "\(userID)" + DemoRepository.ReadSuffixPath
```

Comment:
Doesn't the following work?
`DemoRepository.ReadPrefixPath.appendingPathComponent(String(userID)).appendingPathComponent(DemoRepository.ReadSuffixPath)`.
I'm not sure if this existed for NSString, in that case you could create an extension that transforms it and appends.